schema/extraction_config.schema.json

```diff
@@ -319,6 +319,25 @@
                     "description": "Another filter instance which is applied to the parent node"
                 }
             }
+        },
+        "rebrowse-triggers": {
+            "type": "object",
+            "unevaluatedProperties": false,
+            "description": "Subscribes server namespace nodes to rebrowse which can be filtered by namespaces.",
+            "properties": {
+                "namespace-publication-date": {
+                    "type": "boolean",
+                    "description": "Indicate if the extractor should turn on subscription to namespaces."
```

Comment:
"Indicate if the extractor should subscribe to namespacePublicationDate on the Server node."